### PR TITLE
FINE YOU ASSHOLES I AM SICK AND TIRED OF ALL THIS REVERTS GOOFCHEM

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: IAMSANICBALL and PKSANICGUIN321
rscadd: GOTA GOFAST
/:cl:

4a428504-83ee-4c5c-be28-66c8bf66a65f
sanic_by_daltond-d71lysv
sanic2
200px-sonichuoriginal
300px-0782-cwcspread

### CODE REVIEW

**Comment by Iamgoofball** [comment]
# :heart: TIME ~~~ CAPSULE :heart:
# DON'T TOUCH UNTIL 4/1/16

**Comment by AdamElTablawy** [comment]
what have you done

**Comment by Iamgoofball** [comment]
i sanic sanic now

**Comment by PKPenguin321** [comment]
**THE RITUAL BEGIIIIINS**

**Comment by Aloraydrel** [comment]
MEMES

**Comment by Aloraydrel** [comment]
How long has it been?

**Comment by Archie700** [comment]
rip

**Comment by KorPhaeron** [comment]
Nevermind I just saw sonichu I'm merging this for real

**Comment by PKPenguin321** [comment]
It's for April Fools you madman

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -763,3 +764,23 @@ SYNDICATE BLACK OPS
 	specflags = list(RADIMMUNE,VIRUSIMMUNE,NOBLOOD,PIERCEIMMUNE,EYECOLOR)
 	sexes = 0
 
+
+
+/datum/species/sanic
+	name = "Sanic"
+	id = "sanic"
+	roundstart = 1
+	speedmod = -1
```
Comment (Supermichael777) [inline code/modules/mob/living/carbon/human/species_types.dm]: GOTAGOREALYFAST

Comment (duncathan) [inline code/modules/mob/living/carbon/human/species_types.dm]: Only -1?

**Comment by Niknakflak** [comment]
Actual april fools stuff isn't done through github because it's a surprise for the players, close this shit

**Comment by Steelpoint** [comment]
This is fucking retarded.

**Comment by Dorsisdwarf** [comment]
No fuck off

**Comment by TrustyGun** [comment]
faster
faster
gotta go
# **FASTER**

**Comment by GunHog** [comment]
Bit early for the April memes, eh Goof?

**Comment by freerealestate** [comment]
this is actually pretty decent satire of the shit that's been allowed into the game recently

**Comment by Dorsisdwarf** [comment]
It would be better satire if he wasnt mostly satirising himself.

**Comment by GunHog** [comment]
Goof is usually the one to have a **MAJOR, EXTREME, LEVEL 10 MELTDOWN** when someone attempts to add a new race. Of the opponents, he is one of the saltiest and most outspoken.

So yes, this is entirely self-satire!

**Comment by Dorsisdwarf** [comment]
Oh, I thought you were talking about terrible jokes, bad spriting, direct rips, and shameless memes.

**Comment by PKPenguin321** [comment]
I feel like I should point out that I had the concept and like half the code for this, and I also made the sprites myself, but goofball actually managed to finish and PR it. Don't bully it just because lolbullygoofball

**Comment by Iamgoofball** [comment]
sanichu is anorignal concept dontu steeelkldls

File: code/modules/mob/living/carbon/human/species_types.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/species_types.dm]: mus rticnretat fast

**Comment by Iamgoofball** [comment]
@Niknakflak april fools '15 was open on the github weeks in advance

**Comment by Supermichael777** [comment]
private repository and just holding the code on local for the minimal amount of testing it gets

**Comment by Shadowlight213** [comment]
Yeah, hg had a private repo for the previous April fools.

**Comment by Iamgoofball** [comment]
@Shadowlight213 no, it was a open PR that @xhuis and I worked on, let me go find it

he had a repo that he loaded it on for the purposes of getting it onto the server but that was it

**Comment by Shadowlight213** [comment]
It wasn't just the shittening, it was also tajarans and corgipeople. I think those were older prs though actually.

**Comment by Iamgoofball** [comment]
yup, corgipeople was a previous denied PR and tajarans get PR'd at least once every year(usually by me for slippery slope memes)

**Comment by Xhuis** [comment]
Here's the PR: #8514
And yes, there was a private repo that was used for April Fools. The PR in question was just put onto that repository beforehand - An0n3 told me this afterwards.

**Comment by Shapsy** [comment]
>meme pr causing this much arguing

**Comment by Cheridan** [comment]
i think this has gone too far

**Comment by KorPhaeron** [comment]
fuck off jerkops

**Comment by Aloraydrel** [comment]
manajerk

**Comment by duncathan** [comment]
L M A O
:joy:

**Comment by PKPenguin321** [comment]
my sides dude jesus christ

**Comment by Iamgoofball** [comment]
gpogm typp fasyt

**Comment by Ricotez** [comment]
maxresdefault

**Comment by Niknakflak** [comment]
Please close

**Comment by PKPenguin321** [comment]
No

**Comment by Jazaen** [comment]
KOR YOU LIAR
You had my hopes up ;_;

**Comment by PKPenguin321** [comment]
kor what you have doine is bad

**Comment by Shadowlight213** [comment]
abuse

**Comment by Aloraydrel** [comment]
BETA UPRISING NOW

**Comment by TechnoAlchemisto** [comment]
FUCK

**Comment by Shadowlight213** [comment]
Soooo much abuse

**Comment by PKPenguin321** [comment]
>BETA UPRISING NOW

**REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE**

**Comment by Bawhoppen** [comment]
you autists I cannot believe this

**Comment by Cheridan** [comment]
merge conflict :^)

**Comment by PKPenguin321** [comment]
Fuckkk fix it fix fix fix it

**Comment by Iamgoofball** [comment]
when its done

**Comment by KorPhaeron** [comment]
Open april fools shit when its actually april fools

**Comment by Xhuis** [comment]
The problem with that is that the server would require constant updates for speedmerged content, which is obviously bad. Opening holiday PRs in advance let's them get merged in time for their holidays.

**Pull request closed without merging**